File: src/main/java/no/nav/pensjondokdist/HealthController.java
package no.nav.pensjondokdist;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class HealthController {
    @GetMapping("/api/internal/isReady")
    public String isReady() {
        return "ready";
    }

    @GetMapping("/api/internal/isAlive")
    public String isAlive() {
        return "alive";
    }
}

File: src/main/kotlin/no/nav/pensjon/dokdist/HealthController.kt
package no.nav.pensjon.dokdist

import org.springframework.web.bind.annotation.*

@RestController
class HealthController {
    @GetMapping("/api/internal/isReady")
    fun isReady(): String =
        "ready"

    @GetMapping("/api/internal/isAlive")
    fun isAlive(): String =
        "alive"
}
